makefiles/git.mk

```diff
@@ -3,13 +3,15 @@ $(error Use the main Makefile)
 endif
 
 SUBPROJECTS += git
-GIT_VERSION := 2.31.1
-DEB_GIT_V   ?= $(GIT_VERSION)-1
+GIT_VERSION := 2.32.0
+DEB_GIT_V   ?= $(GIT_VERSION)
 
 GIT_ARGS += uname_S=Darwin \
 	HOST_CPU=$(GNU_HOST_TRIPLE) \
 	DESTDIR=$(BUILD_STAGE)/git \
 	MANDIR=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man \
+	PYTHON_PATH=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/python3 \
+	ASCIIDOC_PATH=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/asciidoc \
```

Comment:
For the two new lines here, is this wanting the host bins or the build bins?

Reply:
It needs the bins that are on the host system. Cause it looks for those bins before it can build.

Reply:
`PYTHON_PATH` is used to set the shebang on scripts.

Reply:
@CRKatri and asciidoc?

Reply:
`ASCIIDOC_PATH` does not exist, but it needs asciidoc to compile the documentation during build.

Reply:
Not sure if I'm understanding this correctly but if Git needs asciidoc to build why is `MEMO_{SUB_}PREFIX` being used to find it? Shouldn't it be `$(which asiidoc)` or something similar?

Reply:
`ASCIIDOC_PATH` does nothing now that I have rebuilt it twice. If you have ASIIDOC installed on the system it will be build the docs regardless of it being a variable. It will be removed now.